Test/Integration/Model/ConfigTest.php

```diff
@@ -20,6 +20,11 @@ protected function setUp(): void
         $this->config = TestFrameworkObjectManager::getInstance()->create(Config::class);
     }
 
+    public function testEuCdnEnabledReturnsFalseByDefault(): void
```

Comment:
Please move this to the bottom to match the order in the other files. Also, please also test if the right value is returned if a value is set.